config/vufind/CookieConsent.yaml

```diff
@@ -0,0 +1,132 @@
+---
+# Cookie consent and IFrame manager configuration.
+#
+# You can use e.g. https://cookiedatabase.org/ to find out the cookies used by most
+# popular services.
+#
+# ControlIframeServices links a category consent with iframe services so that a
+# consent allows the iframe to be loaded without a prompt.
+#
+# ControlVuFindServices links a category consent with VuFind services so that a
+# consent allows the service to be controlled. The following services are currently
+# supported: matomo
+#
+# The following placeholders can be used in settings and translated texts:
+# {{consent_cookie_name}}         - Name of the consent cookie (default cc_cookie)
+# {{consent_cookie_expiration}}   - Consent cookie expiration time
+# {{current_host_name}}           - Server name of the VuFind server
+# {{vufind_cookie_domain}}        - Domain for VuFind's cookies
+# {{vufind_session_cookie}}       - VuFind session cookie name
+CookieConsent:
+  #CookieExpiration: 365
```

Comment:
What is the purpose of this commented-out setting?

Reply:
It can be used to change the expiration for the consent cookie.

Reply:
Makes sense, so I guess we just need to document it. Then the question becomes, do we document it in a big comment at the top of the YAML or in the wiki? I lean toward "big comment" since that ensures that the documentation and configuration file are always in sync. We can use the wiki for example configs and link to the YAML file itself for more detailed documentation (unless you feel otherwise).